website/js/repl.js

```diff
@@ -44,6 +44,12 @@ var optionsConfig = [
     defaultVal: "browser",
     description: "The target environment for Prepack"
   },
+  {
+    type: "string",
+    name: "lazyObjectsRuntime",
+    defaultVal: "",
+    description: "Enable lazy objects feature and specify the JS runtime that support this feature."
```

Comment:
sp: supports

Reply:
Thanks for pointing out. Will commit a fix shortly.